bcda/cclf/cclf.go

```diff
@@ -275,7 +276,23 @@ func importCCLF(ctx context.Context, fileMetadata *cclfFileMetadata, importer im
 			continue
 		}
 
-		err = importer.do(ctx, txn, cclfFile.ID, b)
+		const (
+			mbiStart, mbiEnd   = 0, 11
+			hicnStart, hicnEnd = 11, 22
```

Comment:
Since we don't use HICN anymore, maybe a future ticket is needed to cease ingestion of this data?